domains/gizzy.json

```diff
@@ -1,27 +1,28 @@
-{
-  "description": "gizzy.is-a.dev",
-  "owner": {
-    "username": "GizzyUwU",
-    "email": "nerop455@gmail.com",
-    "discord": "gizzyowo"
-  },
-  "record": {
-    "A": [
-      "185.199.108.153",
-      "185.199.109.153",
-      "185.199.110.153",
-      "185.199.111.153"
-    ],
-    "AAAA": [
-      "2606:50c0:8000::153",
-      "2606:50c0:8001::153",
-      "2606:50c0:8002::153",
-      "2606:50c0:8003::153"
-    ],
-    "MX": [
-      "mx1.forwardemail.net",
-      "mx2.forwardemail.net"
-    ],
-    "TXT": "forward-email=nerop455@gmail.com"
-  }
-}
+{
+  "description": "gizzy.is-a.dev",
+  "owner": {
+    "username": "GizzyUwU",
+    "email": "nerop455@gmail.com",
+    "discord": "gizzyowo"
+  },
+  "record": {
+    "A": [
+      "185.199.108.153",
+      "185.199.109.153",
+      "185.199.110.153",
+      "185.199.111.153",
+      "76.76.21.21"
+    ],
+    "AAAA": [
+      "2606:50c0:8000::153",
+      "2606:50c0:8001::153",
+      "2606:50c0:8002::153",
+      "2606:50c0:8003::153"
+    ],

```

Comment:
Why do you need both GitHub Pages and Vercel IP?

Reply:
stupid mistake meant to do it on api.gizzy :p